api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -555,7 +555,7 @@ func validateManagedClusterNetwork(cli client.Client, labels map[string]string,
 	}
 
 	if err := cli.Get(ctx, key, ownerCluster); err != nil {
-		allErrs = append(allErrs, field.InternalError(field.NewPath("Cluster", "Spec", "ClusterNetwork"), err))
+		allErrs = append(allErrs, field.InternalError(field.NewPath("cluster", "spec", "clusterNetwork"), err))
```

Comment:
I don't think `Cluster` should be changed either, just everything after the `spec`